README.md

```diff
@@ -29,34 +29,34 @@ Linters which are not language-specific:
-| Scala                  | [scalafmt]                |                                  |
-| Shell                  | [shfmt]                   | [shellcheck]                     |
-| Starlark               | [Buildifier]              |                                  |
-| Swift                  | [SwiftFormat] (1)         |                                  |
-| TSX                    | [Prettier]                | [ESLint]                         |
-| TypeScript             | [Prettier]                | [ESLint]                         |
-| YAML                   | [yamlfmt]                 |                                  |
-| XML                    | [prettier/plugin-xml]     |                                  |
+| Language               | Formatter                 | Linter(s)                        | Type checkers (in other repos) |
+| ---------------------- | ------------------------- | -------------------------------- | ------------------------------ |
+| C / C++                | [clang-format]            | [clang-tidy]                     |                                |
+| Cuda                   | [clang-format]            |                                  |                                |
+| CSS, Less, Sass        | [Prettier]                | [Stylelint]                      |                                |
+| Go                     | [gofmt] or [gofumpt]      |                                  |                                |
+| Gherkin                | [prettier-plugin-gherkin] |                                  |                                |
+| GraphQL                | [Prettier]                |                                  |                                |
+| HCL (Hashicorp Config) | [terraform] fmt           |                                  |                                |
+| HTML                   | [Prettier]                |                                  |                                |
+| JSON                   | [Prettier]                |                                  |                                |
+| Java                   | [google-java-format]      | [pmd] , [Checkstyle], [Spotbugs] |                                |
+| JavaScript             | [Prettier]                | [ESLint]                         |                                |
+| Jsonnet                | [jsonnetfmt]              |                                  |                                |
+| Kotlin                 | [ktfmt]                   | [ktlint]                         |                                |
+| Markdown               | [Prettier]                | [Vale]                           |                                |
+| Protocol Buffer        | [buf]                     | [buf lint]                       |                                |
+| Python                 | [ruff]                    | [flake8], [ruff]                 | [mypy]                         |
```

Comment:
I think this makes the table very wide, and can't imagine any other rows that will ever have a value in the new column. Maybe we could treat it more like keep-sorted above, where it's an exception?  or just edit the FAQ at the bottom to point to mypy?

Reply:
Done, added it in the FAQ.